lesson2/node.js

```diff
@@ -0,0 +1,7 @@
+const f = require('fs');
+
+const read = (err,files)=>{
+  console.log(files);
```

Comment:
good start! Now implement part 2:
Print out all the file names that are not you :)